tools/tpm/policy_sign.c

```diff
@@ -205,7 +205,7 @@ static void printHexString(const unsigned char* bin, unsigned long sz,
     printf("\t");
     for (i = 0; i < sz; i++) {
         printf("%02x", bin[i]);
-        if (((i+1) % maxLine) == 0 && i+1 != sz)
+        if (maxLine > 0 && ((i+1) % maxLine) == 0 && i+1 != sz)
```

Comment:
as maxLine is unsigned, this check is always true

Reply:
it will be false if `maxLine == 0` (the check is used to avoid div by zero in `(i+1) % maxLine`), should I change the check in `maxLine != 0` ?

Reply:
my bad, I misread as `>=`